hil/client/node.py

```diff
@@ -32,7 +34,25 @@ def register(self, node, subtype, *args):
         # and currently active drivers for HIL
         # obm_api = "http://schema.massopencloud.org/haas/v0/obm/"
         # obm_types = ["ipmi", "mock"]
-        raise NotImplementedError
+        if (len(args) != 3):
+            raise BadArgumentError
```

Comment:
I think you should tell why it's bad. ("3 Arguments needed. Supplied = len(args)")

Reply:
sure!